### ISSUE TITLE
o-forms: improve error description in demo

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#542]
# Issue
This is from page 129 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=toggle&system_code=origami-repo-data&brand=master>

The error message ‘Something has gone wrong’ is non-descriptive and does not allow users to understand what error has occurred in the toggle inputs. The error message has been placed in a span below the input fields, for sighted users they are able to identify the error message correctly by the colour red but for screen reader users the message is not announced as an alert.

 
# Proposed Solution:

Ensure screen reader users are able to identify when errors have occurred in the checkboxes. In this case, the error message should be more descriptive to inform users what type of error has occurred and assign the role=”alert” to the span which contains the error message, this will enable all users to identify the error message correctly.
Further information can be found in: [Error summary](https://design-system.service.gov.uk/components/error-summary/) and [Error message](https://design-system.service.gov.uk/components/error-message/).

**Comment by akomiqaia** [comment]
Closing this issue. It seems we already fixed it in #828
